src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs

```diff
@@ -484,7 +504,8 @@ private async Task<IActionResult> ExchangeAuthorizationCodeOrRefreshTokenGrantTy
                 // The other claims will only be added to the access_token, which is encrypted when using the default format.
                 if ((claim.Type == OpenIddictConstants.Claims.Name && ticket.HasScope(OpenIddictConstants.Scopes.Profile)) ||
                     (claim.Type == OpenIddictConstants.Claims.Email && ticket.HasScope(OpenIddictConstants.Scopes.Email)) ||
-                    (claim.Type == OpenIddictConstants.Claims.Role && ticket.HasScope(OpenIddictConstants.Claims.Roles)))
```

Comment:
It's worth noting that this logic is not really effective, as we don't manually add these claims anywhere. I assigned https://github.com/OrchardCMS/OrchardCore/issues/812 a milestone so we can take a look at a later date).